profiles/killbill/pom.xml

```diff
@@ -284,6 +284,7 @@
         <dependency>
             <groupId>org.kill-bill.billing</groupId>
             <artifactId>killbill-client-java</artifactId>
+            <version>1.3.0-24f1d97-SNAPSHOT</version>
```

Comment:
Could you override the property in the top level pom.xml instead?

In the `<properties>` section:

```
<killbill-client.version>1.3.0-24f1d97-SNAPSHOT</killbill-client.version>
```

Reply:
I already moved the client version